src/fu-common-cab.c

```diff
@@ -227,12 +227,49 @@ fu_common_store_from_cab_folder (AsStore *store, GCabFolder *cabfolder, GError *
 	return TRUE;
 }
 
+typedef struct {
+	guint64		 size_total;
+	guint64		 size_max;
+	GError		*error;
+} FooHelper;
```

Comment:
Not really a fan of this name.  How about something like CabCBHelper or something more descriptive?

Reply:
Gahh! I was supposed to change that, good catch. Fixed.